### ISSUE TITLE
Determine what copyrights can be asserted over the NFT images we create

### ISSUE THREAD

**Issue by inverted-capital** [dreamcatcher-tech/dreamcatcher-tech.github.io#18]
If we own the copyright on the generated NFT images, we can use copyright to force advertising for the Dreamcatcher.  For example, we might forbid reproduction of the image unless a reference to each NFTs homepage is provided, which shows who the current owners are, links to the containing pool, and to the Dreamcatcher.

This also adds to the exclusivity of each NFT, as well as adding to the bundle of value that is purchased by anyone who owns an NFT.  The bundle is a little thin until Ambient Attribution is realized, so we need to pack it with whatever we can manage !

**Comment by TemperBead** [comment]
Here's the relevant section from hotpot.ai:  https://hotpot.ai/terms

TLDR: 

Purchasing an NFT pic through them is a grant of license for use by the purchaser, not a transfer of title.  Specifically, it does not grant transfer of that license for the content to another person without their permission.  They also reserve the right to use the content themselves, but it's unclear to me whether this is just for their 'make it public' function or for everything.  They also don't guarantee uniqueness:  "Because multiple users may share similar ideas, it is possible some users may use the Services and create identical or similar output."  Which does make sense. 

That being said, their terms aren't well written.  I've just emailed them asking for clarifications about what the license actually permits (they only list what it does not permit), the transfer of that license and whether there's any obligation to display originators copyright information.

**Comment by TemperBead** [comment]
Looking better for OpenAI API output:  

"2 (c) Copyright. OpenAI will not assert copyright over Content generated by the API for you or your end users."

https://openai.com/api/policies/terms/

**Comment by inverted-capital** [comment]
> Looking better for OpenAI API output:
> 
> "2 (c) Copyright. OpenAI will not assert copyright over Content generated by the API for you or your end users."
> 
> https://openai.com/api/policies/terms/

I'm on the waitlist for DALL-E but could be a while till they're operating.  Setting up our own generators might be an option.  I think we can close this ticket if we just say we'll assert maximum possible copyright over the images, to add to the value of the NFT, and accept whatever rules the generation services put on us ?

**Comment by TemperBead** [comment]
Agreed.  Ideally when we get to that stage we just put up a Request for an image generator, then perhaps ping a few people e.g. like those involved in this repo:  https://github.com/lucidrains/DALLE2-pytorch
